_data/retail.yml

```diff
@@ -412,6 +406,12 @@ websites:
     twitter: pccasegear
     img: pccasegear.png
 
+  - name: Printique
+    url: https://www.printique.com
+    img: printique.png
+    twitter: Printique_lab
+    facebook: PrintiqueLLC
+
```

Comment:
Printique only serves US & Canada (https://www.printique.com/company/shipping-processing/).

```
regions:
  - us
  - ca
```
(It's not letting me add a suggestion to the code for some reason 😕)